java-checks/src/test/java/org/sonar/java/checks/DepthOfInheritanceTreeCheckTest.java

```diff
@@ -22,50 +22,53 @@
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.JavaCheckVerifier;
 
+import static org.sonar.java.checks.verifier.TestUtils.nonCompilingTestSourcesPath;
 import static org.sonar.java.checks.verifier.TestUtils.testSourcesPath;
 
-class DITCheckTest {
+class DepthOfInheritanceTreeCheckTest {
 
   @Test
   void defaults() {
     JavaCheckVerifier.newVerifier()
-      .onFile("src/test/files/checks/DitOk.java")
```

Comment:
I would also rename the test files to contain the name of the check in their name.
```
DitOk.java -> DepthOfInheritanceTreeCheckOk.java
``` 
This comment applies to other files as well.